src/Polytopes/Polyhedron/properties.jl

```diff
@@ -140,88 +144,82 @@ end
 Base.eltype(::Type{VertexPointIterator}) = Polymake.Vector{Polymake.Rational}
 Base.length(iter::VertexPointIterator) = nvertices(iter.p)
 
-"""
-    vertices(as, P)
+@doc Markdown.doc"""
+    vertices(as::Type{T} = Points, P::Polyhedron)
 
-Return an iterator over the vertices of a polyhedron `P` in the format defined by `as`.
-Optional arguments for `as` include
-* `Points`: Returns the representation of a vertex as a point.
+Return an iterator over the vertices of `P` in the format defined by `as`.
 
-See also `vertices_as_point_matrix`.
+Optional arguments for `as` include
+* `Points`.
 
-# Arguments
-- `as::Type{T}`: Element type of the returned iterator.
-- `P::Polyhedron`: A polyhedron.
+See also: [`vertices_as_point_matrix`](@ref).
 
 # Examples
-The following code computes the vertices of the Minkowski sum of a triangle and a square:
-```julia-repl
+The following code computes the vertices of the Minkowski sum of a triangle and
+a square:
+```jldoctest
 julia> P = simplex(2) + cube(2);
 
 julia> collect(vertices(Points, P))
 5-element Vector{Polymake.Vector{Polymake.Rational}}:
-pm::Vector<pm::Rational>
+ pm::Vector<pm::Rational>
 -1 -1
-pm::Vector<pm::Rational>
+ pm::Vector<pm::Rational>
 2 -1
-pm::Vector<pm::Rational>
+ pm::Vector<pm::Rational>
 2 1
-pm::Vector<pm::Rational>
+ pm::Vector<pm::Rational>
 -1 2
-pm::Vector<pm::Rational>
+ pm::Vector<pm::Rational>
 1 2
 ```
 """
 function vertices(as::Type{T}, P::Polyhedron) where {T}
     if as == Points
         VertexPointIterator(P)
     else
-        throw(ArgumentError("Unsupported `as` argument :" * string(as)))
+        throw(ArgumentError("Unsupported `as` argument:" * string(as)))
```

Comment:
If you change one of them, perhaps change all? ;-).
```suggestion
        throw(ArgumentError("Unsupported `as` argument: $(as)"))
```